008_team_6_agent_design/008_team_6_agent_design.tex

```diff
@@ -1 +1,1014 @@
+    \includegraphics[width=0.3\linewidth]{008_team_6_agent_design/D/D3_utility.pdf}\\[0pt]
+    (d) D2: social motives & (e) D2: deaths over time & (f) D2: utility over time \\[8pt]
+    \end{tabular}
+    \caption{Simulation results different treaties acceptances.}
+    \label{fig:res_D}%
+\end{figure}
+
+The analysis of experiment D1 can be done as follows: during the first days in the tower, the agents will agree on collectivist treaties as introduced in \Cref{proposing_treaties}. As the turns progress and their health decreases, some agents change their social motive from collectivist to selfish (\Cref{fig:res_D} (a)). However, once their health falls below the \textit{weakLevel}, the treaties the agents signed whilst being collectivist no longer apply. This is because, as explained in \Cref{proposing_treaties}, the treaties proposed by collectivists only hold under the condition \textit{currentHP} $>$ \textit{WeakLevel}. Therefore, the selfish agents, now unconstrained, consume more food than what a collectivist agent normally would. This wastes utility, ultimately leading to a large amount of deaths at the end of day 10 (\Cref{fig:res_D} (b)). This result is similar to the one discussed in \Cref{simulation_C}.
+
+The waste of common pool resources can also be visualized in \Cref{fig:res_D} (c), where the global utility becomes negative on day 10. We attribute the significance of the spikes at day 10 to its correspondence to the number of days an agent can survive without eating.
+
+After 10 days, the terminated agents are replaced by a new group of collectivists agents, leading to a peak in \Cref{fig:res_D} (a). During days 11 to 20, no deaths are seen (\Cref{fig:res_D} (d)), as all selfish agents are still following the treaty they signed when collectivist. Note that the new agents introduced on day 11 sign the collectivist treaties and will not eat any food if their health is above the \textit{weakLevel}, even if they become selfish. On day 20, we note a new peak in the number of deaths, and this process repeats itself. This results in oscillatory behaviour between 2 quasi-stable states, with convergence to both a high concentration of collectivists and selfish agents in inverse proportions. We hence deem this experiment as 2-phase polystable \cite{ashby2015design}.
+
+% In addition to the treaty used in experiment D1, the agents have the ability to propose and accept the following treaty in experiment D3 \ToDo{rename D3 to D2 and drop the current D2 experiment which is redundant}: 
+
+
+In order to avoid having the selfish agents waste the common pool resources when their health drops below the \textit{weakLevel}, we enforce the agents to sign another, more restrictive treaty during the first days of the simulation as in D3 (\Cref{simulation_C}). We note two interesting results when using this additional treaty.
```

Comment:
```suggestion
In order to avoid having the selfish agents waste the common pool resources when their health drops below the \texttt{weakLevel}, we enforce the agents to sign another, more restrictive treaty during the first days of the simulation as in D3 (\Cref{simulation_C}). We note two interesting results when using this additional treaty.
```